charts/ceph-csi-rbd/values.yaml

```diff
@@ -142,7 +142,7 @@ provisioner:
   provisioner:
     image:
       repository: quay.io/k8scsi/csi-provisioner
-      tag: v1.4.0
+      tag: v1.6.0
```

Comment:
this will break the snapshot. we need to wait till we move snapshot to beta in ceph-csi . as we are using 1.4.0 we are fine